api/src/apps/hooks.rs

```diff
@@ -117,7 +118,7 @@ impl Apps {
             .collect::<Vec<_>>();
 
         let js_configs = serde_json::to_value(js_configs).expect("Should be serializable");
-        let js_configs = Value::from_json(js_configs, context);
+        let js_configs = JsValue::from_json(&js_configs, context).unwrap();
```

Comment:
We should handle the error case and maybe bubble that up.

Reply:
Error case is handled with below line
let js_configs =
            JsValue::from_json(&js_configs, &mut context).expect("Unable to read JSON value");